src/LengthUnit.php

```diff
@@ -172,4 +172,48 @@ interface LengthUnit
         self::VMIN,
         self::VMAX,
     ];
+
+    /**
+     * Pixels per inch resolution to use for conversions.
+     *
+     * @var int
+     */
+    const PPI = 96;
+
+    /**
+     * Centimeters per inch.
+     *
+     * @var float;
+     */
+    const CM_PER_IN = 2.54;
+
+    /**
+     * Convert a unit-based length into a number of pixels.
+     *
+     * @param int|float $value Value to convert.
+     * @param string    $unit  Unit of the value.
+     * @return int|float|false Converted value, or false if it could not be converted.
+     */
+    public static function convertIntoPixels($value, $unit)
+    {
```

Comment:
Special case since `0` is `0` regardless of unit.

```suggestion
    {
        if ( 0 === $value ) {
            return 0;
        }
```